app/policies/analytic_policy.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+# User Policy
+class AnalyticPolicy < Struct.new(:user, :analytic)
```

Comment:
Be sure to account for CodeClimate issues.  This includes using `#rubocop:disable` if needed.